internal/program/program.go

```diff
@@ -89,6 +92,13 @@ func (p *Program) Run() error {
 			"Version: %s\nCommit: %s\nDate: %s\n",
 			p.version, p.commit, p.date,
 		)
+		if p.modVersion != "" && err != nil {
```

Comment:
I think I'm missing something about the `err != nil` part: does it mean that the module info should be printed only if printing the regular version failed? Maybe it would be better to build the final string format with its arguments separately, and then print with a single `Fprintf` call?